pkg/csi/service/vanilla/controller.go

```diff
@@ -214,6 +215,15 @@ func (c *controller) Init(config *cnsconfig.Config, version string) error {
 			return err
 		}
 	}
+	if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.CSIVolumeManagerIdempotency) {
```

Comment:
Is this CR only applicable to controller APIs or Node APIs too? Don't we need to initialize this in node.go as well?

Reply:
Only for controller. Specifically in Volume Manager before making calls to CNS.